.github/actions/gradle/download/action.yml

```diff
@@ -19,8 +22,13 @@ runs:
           curlOpts+=(--header 'Authorization: Bearer ${{ inputs.token }}')
         fi
 
+        releaseUrl="https://api.github.com/repos/gradle/gradle-enterprise-build-validation-scripts/releases/latest"
+        if [ ! -z "${{ inputs.version }}" ]; then
+          releaseUrl="https://api.github.com/repos/gradle/gradle-enterprise-build-validation-scripts/releases/tags/${{ inputs.version }}"
+        fi
+
         # Get the download url of the latest release
```

Comment:
The comment should be adjusted (it's not always getting the latest release)

Reply:
Good point. I've updated this in the latest commit. 831909bfd1b9ce6fc7306ad65e51953f06519b3a